app/ai/page.tsx

```
@@ -0,0 +1,185 @@
+              <div>
+                <h3 className="text-2xl font-semibold text-foreground mb-4">Natural Language to Code</h3>
+                <p className="text-muted-foreground mb-6">Input your requirements, get optimized TEAL or PyTeal code instantly.</p>
+                <ul className="space-y-2 text-left">
+                  <li className="flex items-center gap-2 text-primary"><Zap className="w-4 h-4" /> Supports TEAL, PyTeal, and more</li>
+                  <li className="flex items-center gap-2 text-primary"><Zap className="w-4 h-4" /> Context-aware suggestions</li>
+                  <li className="flex items-center gap-2 text-primary"><Zap className="w-4 h-4" /> Auto-complete and refactoring</li>
+                </ul>
+              </div>
+              <div className="text-center">
+                <div className="bg-muted rounded-lg p-6 mx-auto max-w-sm">
+                  <pre className="text-sm font-mono text-foreground">/* AI Generated */
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)